README.md

```diff
@@ -109,66 +109,83 @@ You can see the result in [Caragua](https://rawgit.com/TerraME/publish/master/ex
-    and it is carried out without any forest management.]])
-
-report:addText([[Welfare indicators varies from regular to satisfactory, and the interviewees declared positive perception of security,
-    housing, participation in the decision-making, leisure activities, festivities, solidarity and equitable division of tasks between men and women.
-    The most frequent demands are for health and education services, water supply, followed by access to electric energy, telephony, Internet, and
-    institutional support to implement new activities for income generation. After this preliminary report, the collected data will be organized in a
-    geographical database to analyze settlements networks.Integrating these data with information collected from previous fieldworks will contribute to
-    better understand the role of settlement networks as part of urban tissue in the southwest of Pará state.]])
+local description = [[
+    This report presents the methodology and the initial results obtained at the fieldwork along riverine settlements at
+    Arapiuns River, tributary of Tapajós River, municipality of Santarém, Pará state, from June 4 th to 15 th , 2012.
+    This research reproduces and extends the data collection accomplished for Tapajós communities in 2009, regarding the
+    infrastructure and network relations of riverine human settlements. The main objective was to characterize the
+    organization and interdependence between settlements concerning to:infrastructure, health and education services,
+    land use, ecosystem services provision and perception of welfare. Escada et. al (2013).
+]]
 
 Application{
     project = filePath("arapiuns.tview", "publish"),
+    description = description,
     base = "roadmap",
     clean = true,
     output = "ArapiunsWebMap",
-    report = report,
     template = {navbar = "#00587A", title = "#FFFFFF"},
     beginning = View{
         description = "Route on the Arapiuns River.",
         width = 3,
-        color = "orange",
-        border = "orange"
+        color = "mediumblue",
+        border = "mediumblue"
     },
     ending = View{
         description = "Route on the Arapiuns River.",
         width = 3,
-        color = "orange",
-        border = "orange"
+        color = "red",
+        border = "red"
     },
     villages = View{
+        download = true,
         description = "Riverine settlements corresponded to Indian tribes, villages, and communities that are inserted into public lands.",
+        icon = "home",
+        select = "Nome",
+        report = function(cell)
+            local mreport = Report{
+                title = cell.Nome,
+                author = "Escada et. al (2013)"
+            }
+
+            mreport:addImage(packageInfo("publish").data.."arapiuns/"..cell.Nome..".jpg")
+
+            local has = "has"
+            local hasnt = "hasn't"
```

Comment:
Just to make the script shorter, use these values when necessary and remove these two local variables.